{"uuid": "649a3851-47b1-4c57-acb9-839b992abbee", "children": ["90946a86-b903-49cf-999f-a1236e0c07ab", "26251b5c-e104-4b59-b7f2-94ab0258d26b", "906ccb81-4ea0-4e7a-b89d-7d6233549702", "fb6b4b42-2794-45cc-9a63-6d27bcb817f6"], "befores": [{"name": "setup", "status": "passed", "start": 1693481940086, "stop": 1693481970172}], "afters": [{"name": "setup::0", "status": "broken", "statusDetails": {"message": "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=52892): Max retries exceeded with url: /session/bac1197e06cff61fb9b3c6924ba39d74/window (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ccf5a50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n", "trace": "  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 911, in _teardown_yield_fixture\n    next(it)\n  File \"/Users/megha/PycharmProject/pywebautomationframework/tests/vwo_login/conftest.py\", line 15, in setup\n    driver.close()\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 456, in close\n    self.execute(Command.CLOSE)\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 343, in execute\n    response = self.command_executor.execute(driver_command, params)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/selenium/webdriver/remote/remote_connection.py\", line 291, in execute\n    return self._request(command_info[0], url, body=data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/selenium/webdriver/remote/remote_connection.py\", line 312, in _request\n    response = self._conn.request(method, url, body=body, headers=headers)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/urllib3/_request_methods.py\", line 110, in request\n    return self.request_encode_url(\n           ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/urllib3/_request_methods.py\", line 143, in request_encode_url\n    return self.urlopen(method, url, **extra_kw)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/urllib3/poolmanager.py\", line 443, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/urllib3/connectionpool.py\", line 874, in urlopen\n    return self.urlopen(\n           ^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/urllib3/connectionpool.py\", line 874, in urlopen\n    return self.urlopen(\n           ^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/urllib3/connectionpool.py\", line 874, in urlopen\n    return self.urlopen(\n           ^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/urllib3/connectionpool.py\", line 844, in urlopen\n    retries = retries.increment(\n              ^^^^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/urllib3/util/retry.py\", line 515, in increment\n    raise MaxRetryError(_pool, url, reason) from reason  # type: ignore[arg-type]\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1693481994809, "stop": 1693481994980}], "start": 1693481940085, "stop": 1693481994981}